demo/index.html

```diff
@@ -98,22 +99,31 @@
           properties: {
             historyElementsAdded: {
               type: Number
+            },
+
+            dwellTime: {
+              type: Number,
+              value: 2000
             }
           },
           observers: [
             'checkHistorySize(path, hash, query, startingHistoryLength)'
           ],
-          ready: function() {
+
+          attached: function() {
```

Comment:
why we set this on `attached`? Does `window.history` change between `ready` and `attached`?